cli-wrap/src/main/resources/cli/listings.sh

```diff
@@ -285,21 +387,41 @@ scm_report() {
 
 scm_report_adv() {
     local type="pretty"
-    if [ "X$1" = "X--json" ]; then
-        type="json"
-        shift
-    fi
-
+    local productNames=""
+    local productVersionIds=""
     local scm="$1"
     local tag="$2"
     local pom_path="$3"
+   
```

Comment:
what about  ``shift 3`` after this, so you don't go over $1, $2 and $3 in the while loop?